docs/api/security/security.md

```diff
@@ -2,10 +2,14 @@
 
```

Comment:
I left a note on a file below about moving its content up to this page.

Reply:
Updated as suggested.